[project-base] - add posibility to override admin styles (#1472)
shopsys/shopsys: upgrade/UPGRADE-v8.1.0-dev.md

        +        Assert::assertNotSame($errorIdFirstAccess, $errorIdSecondAccess);
        +    }
        ```
- add possibility to override admin styles from project-base
 ([#1472](https://github.com/shopsys/shopsys/pull/1472))

>     - delete all files from `project-base/src/Shopsys/ShopBundle/styles/admin/` and create two new files in it - `main.less` and `todo.less` (this file will be empty)

Reviewer: ```suggestion
    - delete all files from `src/Shopsys/ShopBundle/styles/admin/` and create two new files in it - `main.less` and `todo.less` (this file will be empty as it should be used for temporary styles)
```

>     main.less file content:

Reviewer: ```suggestion
    - main.less file content:
```

>     ```diff

Reviewer: You don't need to use `diff` here but I think that you could use `less` so it could be highlighted properly

Author: Less did not work. Used css and it highlight code properly. Thx for this note.

>     // load main file from framework, variable adminCssPath is changed in gruntfile.js

Reviewer: ```suggestion
    // load main.less file from framework, variable adminCssPath is set in gruntfile.js
```

    @import "@{adminCssPath}/main.less";

>     // programmer place for style

Reviewer: ```suggestion
    // file for temporary styles eg. added by a programmer
```

    @import "todo.less";
    ```

>     update `project-base/src/Shopsys/ShopBundle/Resources/views/Grunt/gruntfile.js.twig`

Reviewer: ```suggestion
    - update `src/Shopsys/ShopBundle/Resources/views/Grunt/gruntfile.js.twig`
```
